charts/incubator/lancache-monolithic/questions.yaml

```diff
@@ -81,7 +81,7 @@ questions:
                             description: "This port exposes the container port on the service"
                             schema:
                               type: int
-                              default: 10279
+                              default: 80
```

Comment:
Shouldn't change the external port either @stavros-k

Reply:
Lol you saw at the same time, you're so fast haha

Reply:
lancache cant ever work if its not opened on port 80 and 443. 

You it uses dns to redirect to cache so you can't change the ports. What ever port the client (steam) uses needs to be opened on the node.

Reply:
You can do it when installing. But default won't be 80 or 443.
I don't want users locked out of their system, by just installing an app

Reply:
Okay I get it. But it should be more clear to the user.